webapp/src/components/modals/create_issue/create_issue.jsx

```diff
@@ -16,6 +19,9 @@ const initialState = {
     repoValue: '',
     issueTitle: '',
     issueDescription: '',
+    labels: [],
+    assignees: [],
+    milestone: 0,
```

Comment:
@utkuufuk Thanks for addressing the strategy of storing the user selected values :+1:

One last request from me. Can we make the default value for milestone `null` instead of `0`? I realize `0` is falsy, but it's not clear that it represents the "empty" value. Setting the initial value to `null`, and subsequently the value we set when user has deselected the milestone, makes it clear that the value is empty.